gpu-operator/platform-support.rst

```diff
@@ -323,11 +323,14 @@ The GPU Operator has been validated in the following scenarios:
     .. _ubuntu-kernel:
 
     :sup:`1`
-    For Ubuntu 22.04 LTS, kernel version 5.15 is an LTS ESM kernel.
+    For Ubuntu 22.04 LTS, kernel versions 6.5 and 5.15 are LTS ESM kernels.
     For Ubuntu 20.04 LTS, kernel versions 5.4 and 5.15 are LTS ESM kernels.
     The GPU Driver containers support these Linux kernels.
     Refer to the Kernel release schedule on Canonical's
     `Ubuntu kernel lifecycle and enablement stack <https://ubuntu.com/kernel/lifecycle>`_ page for more information.
+
+    For Ubuntu 22.04 LTS, the GPU Driver containers also support the 6.2 kernel.
```

Comment:
Actually, we should not call out support for the 6.x kernels with GPU Operator 24.3.0 since the driver container fix for the gcc issues has not made it into an official driver container release.

Apologies for the confusion here, we should not have kept the JIRA associated with this in the 24.3.0 initiative. Let's rollback the docs changes you made in e02077c1c776e081b8ae7bf086cdcad843948d88